src/main/g8/pom.xml

```diff
@@ -6,20 +6,16 @@
     <artifactId>app</artifactId>
     <version>1.0</version>
 
-    <properties>
-        <akkaVersion>2.5.1</akkaVersion>
-    </properties>
-
     <dependencies>
         <dependency>
             <groupId>com.typesafe.akka</groupId>
             <artifactId>akka-actor_2.12</artifactId>
-            <version>${akkaVersion}</version>
+            <version>2.5.1</version>
```

Comment:
there is still some problem with using such maven property when using g8, so I simply removed it for now